browser/js/films/FilmFactory.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
Branch of Branch (Film Factory)
Once branch 'Film Factory' is merged, this will not be a change commit.
(i.e. Can ignore this file)